src/main/java/it/gov/pagopa/pu/bff/mapper/PersonalisationFE2ConfigFEMapper.java

```diff
@@ -0,0 +1,27 @@
+package it.gov.pagopa.pu.bff.mapper;
+
+import it.gov.pagopa.pu.bff.dto.generated.ConfigFE;
+import it.gov.pagopa.pu.p4pa_organization.dto.generated.PersonalisationFe;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PersonalisationFE2ConfigFEMapper {
+
+  public ConfigFE mapPersonalisationFE2ConfigFE(PersonalisationFe personalisationFe) {
+    if(personalisationFe == null){
+      return null;
+    }
+    return ConfigFE.builder()
+      .canManageUsers(null)
+      .brokerId(null)
```

Comment:
put a TODO to inform about this fix:
https://pagopa.atlassian.net/browse/P4ADEV-1753

Reply:
added todo

Reply:
put on the TODO also the id of the ticket